crypto/keys/src/ristretto.rs

```diff
@@ -233,6 +237,10 @@ impl RistrettoPublic {
     pub fn to_bytes(&self) -> [u8; 32] {
         self.0.compress().to_bytes()
     }
+
+    pub fn point(&self) -> RistrettoPoint {
+        self.0
+    }
```

Comment:
We already have an `AsRef<RistrettoPoint>`, please don't add redundant/non-generic API to this, or use it in production code.

Reply:
Fixed!